factory/consensus/consensusComponents.go

```diff
@@ -44,22 +44,23 @@ type ConsensusComponentsFactoryArgs struct {
 	ScheduledProcessor    consensus.ScheduledProcessor
 	IsInImportMode        bool
 	ShouldDisableWatchdog bool
+	SubRoundBlockType     consensus.SubRoundBlockType
```

Comment:
Maybe we can keep subround, as a single word, without caps R everywhere in this file